x/gov/types/v1/proposal.go

```diff
@@ -94,17 +94,6 @@ func ProposalStatusFromString(str string) (ProposalStatus, error) {
 	return ProposalStatus(num), nil
 }
 
-// Marshal needed for protobuf compatibility
-func (status ProposalStatus) Marshal() ([]byte, error) {
-	return []byte{byte(status)}, nil
-}
-
-// Unmarshal needed for protobuf compatibility
-func (status *ProposalStatus) Unmarshal(data []byte) error {
```

Comment:
there's also the same method in v1beta1. And also for voteOption. So there are 4 instances to remove in total